lib/ddtrace/contrib/rails/action_controller_patch.rb

```diff
@@ -57,7 +57,7 @@ def process_action(*args)
             end
           # rubocop:enable Lint/RescueException
           ensure
-            Datadog::Contrib::Rails::ActionController.finish_processing(payload)
+            Datadog::Contrib::Rails::ActionController.finish_processing(request.env, payload)
```

Comment:
Could we add this `request.env` to the `payload` object here instead of modifying the method definition? It's used as a generic grab-bag of context information anyways.

Reply:
:+1: